### PR TITLE
OSDOCS-1650: Adding docs for managing user oauth tokens

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1650

Preview: https://osdocs-1650--ocpdocs.netlify.app/openshift-enterprise/latest/authentication/managing-oauth-access-tokens.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sttts I made a few adjustments based on our slack conversation if you can take another look. Especially the wording around oc logout, if that's any better. Thanks!

@rhpmali Can you also please review? Thanks!

Preview: https://osdocs-1650--ocpdocs.netlify.app/openshift-enterprise/latest/authentication/managing-oauth-access-tokens.html

File: modules/oauth-delete-tokens.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-oauth-access-tokens.adoc
+
+[id="oauth-delete-tokens_{context}"]
+= Deleting user OAuth access tokens
+
+The `oc logout` command only invalidates the OAuth token for the active session. You can use the following procedure to delete any OAuth tokens that are no longer needed.
+
+Deleting an OAuth access token logs out the user from all sessions that are using the token.
```
Comment (kalexand-rh) [inline modules/oauth-delete-tokens.adoc:10]:
```suggestion
Deleting an OAuth access token logs out the user from all sessions that use the token.
```

File: modules/oauth-delete-tokens.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.Procedure
+
+* Delete the user OAuth access token.
```
Comment (kalexand-rh) [inline modules/oauth-delete-tokens.adoc:14]:
```suggestion
* Delete the user OAuth access token:
```

File: authentication/managing-oauth-access-tokens.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[id="managing-oauth-access-tokens"]
+= Managing user OAuth access tokens
+include::modules/common-attributes.adoc[]
+:context: managing-oauth-access-tokens
+
+toc::[]
+
+Users can manage their own OAuth access tokens.
```
Comment (kalexand-rh) [inline authentication/managing-oauth-access-tokens.adoc:8]: Would a bit of context about actions that users can't do themselves or what an admin must have done before a user can do anything be helpful here?

**Review by kalexand-rh — COMMENTED** [review]
This is looking good!

File: authentication/managing-oauth-access-tokens.adoc (hunk 3 above)
Comment (bergerhoffer) [reply authentication/managing-oauth-access-tokens.adoc:8]: Not that I can think of. Pretty much users can request their own tokens, and then now they can actually see and delete them if they wanted to. Admins can just manage other users' tokens, but that's using a different resource. Nothing that admins should need to do in advance. But I'll see if I can give any more detail here, it is pretty minimal.

File: modules/oauth-list-tokens.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-oauth-access-tokens.adoc
+
+[id="oauth-list-tokens_{context}"]
+= Listing user OAuth access tokens
+
+You can list all of your user OAuth access tokens. Token names are not sensitive and cannot be used to log in.
+
+.Procedure
```
Comment (stlaz) [inline modules/oauth-list-tokens.adoc:10]: Please also document the possibility of listing tokens per given oauth client (that's effectively listing them per the application the tokens were issued for)

File: authentication/managing-oauth-access-tokens.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[id="managing-oauth-access-tokens"]
+= Managing user OAuth access tokens
```
Comment (stlaz) [inline authentication/managing-oauth-access-tokens.adoc:2]: I would call them user-owned OAuth access tokens

**Comment by bergerhoffer** [comment]
Thanks @stlaz, I've updated per your comments. Let me know if the listing per client is what you were expecting.

@rhpmali Can you please QE review this? Thanks!

https://osdocs-1650--ocpdocs.netlify.app/openshift-enterprise/latest/authentication/managing-oauth-access-tokens.html

**Pull request merged**